readthedocs/restapi/views/footer_views.py

```diff
@@ -75,6 +75,10 @@ def footer_html(request):
     subproject = request.GET.get('subproject', False)
     source_suffix = request.GET.get('source_suffix', '.rst')
 
+    # Hack in a fix for missing version slug deploy that went out a while back
+    if version_slug == '':
+        version_slug = 'latest'
```

Comment:
Maybe this is better,

```
if not version_slug:
    version_slug = LATEST
```

The "linting" guy :)

Reply:
We actually want to check it being an empty string, not None, because that is the error I'm catching.